mkdocs.yml

```diff
@@ -201,36 +201,43 @@ plugins:
         # vi: Vietnamese
       nav_translations:
         de:
+          Home: Startseite
           Get started: Erste Schritte
           How-to guides: Tutorial
           Technical reference: Technische Referenz
           Success stories: Erfolgsgeschichten
         fr:
+          Home: Accueil
           Get started: Mise en route
           How-to guides: Tutoriels
           Technical reference: Référence technique
           Success stories: Témoignages de projets
         it:
+          Home: Casa
```

Comment:
@domi4484 is this ok for "home page"?

Reply:
No it sounds very strange :smile:, `Pagina iniziale` is more appropriated

Reply:
Thank you for checking my translations. It is not easy for me to choose an appropriate word even I use translation software.

Reply:
I fixed translation to `Pagina iniziale`